modules/comments/components/ExecutiveComments.tsx

```diff
@@ -0,0 +1,105 @@
+import { Flex, Text, Box } from 'theme-ui';
+import { useMemo, useState } from 'react';
+import BigNumber from 'bignumber.js';
+
+import Stack from 'modules/app/components/layout/layouts/Stack';
+import { Proposal } from '../../executive/types';
+import FilterButton from 'modules/app/components/FilterButton';
+import { MenuItem } from '@reach/menu-button';
+import { ExecutiveCommentsAPIResponseItem } from '../types/comments';
+import CommentItem from './CommentItem';
+
+export default function ExecutiveComments({
+  proposal,
+  comments,
+  ...props
+}: {
+  proposal: Proposal;
+  comments: ExecutiveCommentsAPIResponseItem[] | undefined;
+}): JSX.Element {
+  const [commentSortBy, setCommentSortBy] = useState('latest');
+  const sortedComments = useMemo(() => {
```

Comment:
this can be optimized and shared between the two lists as a different component in the future, for now keeping it simple